app/assets/javascripts/select-kit/components/user-notifications-dropdown.js.es6

```diff
@@ -73,13 +74,14 @@ export default DropdownSelectBox.extend({
         .catch(popupAjaxError);
     },
     ignore() {
-      this.get("updateNotificationLevel")("ignore")
-        .then(() => {
-          this.set("user.ignored", true);
-          this.set("user.muted", false);
+      const controller = showModal("ignore-duration", {
+        model: this.get("user")
+      });
+      controller.setProperties({
+        refreshHeaderContent: () => {
```

Comment:
Why is this a property and not just a function?

Reply:
So I need to be a property that's containing a function because the `modal` controller doesn't have access to: `computeHeaderContent` of the current component.

What's your suggestion?